src/features/activities/replies/RepliesBox.tsx

```diff
@@ -16,9 +16,60 @@ import { ConfirmationModal } from 'components/ConfirmationModal';
 import { Text } from 'ui';
 
 import { ReplyForm } from './ReplyForm';
+import { ReplyReactionBar } from './ReplyReactionBar';
 
 export const QUERY_KEY_REPLIES = 'query-key-replies';
 
+const fetchReplies = async (activityId: number) => {
```

Comment:
moved here to be able to export type.
it doesn't add value if it's inside the function